fire/api/firedb.py

```diff
@@ -54,10 +54,17 @@ def __init__(self, connectionstring=None, debug=False):
         else:
             self.connectionstring = self._build_connection_string()
 
+        # Kører vi under test setup?
+        if self.connectionstring.startswith("fire.fire@localhost"):
+            exe_opt = {}
```

Comment:
Det er smartere at håndtere test-situationen i `test.conftest.TestFireDb`. For det første er det der koden hører til, for det andet slipper du for at gætte en connectionstring der nemt kan være forskellig fra system til system.

Reply:
Så du mener at jeg skal overstyre FireDbs __init__ i TestFireDb? Det kunne man vel godt, men det ville give en hel del kodeduplikering

Reply:
Ja, det mener jeg og jeg har svært ved at se hvordan det skal kaste en masse kodeduplikering af sig. Lav en `self.exe_opt` og definer den med hver sit indhold i `__init__()` i de to klasser. Start motoren med

```
self.engine = create_engine(
            f"{self.dialect}://{self.connectionstring}",
            connect_args={"encoding": "UTF-8", "nencoding": "UTF-8"},
            echo=debug,
            execution_options=self.exe_opt,
        )
```

Det giver vel omtrent samme antal linjer som du har nu bare uden en if der antager noget om opsætningen på maskinen hvor testsuiten afvikles.